sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -324,9 +324,11 @@ public partial class AnalyzeTextOptions
     {
         public AnalyzeTextOptions(string text) { }
         public AnalyzeTextOptions(string text, Azure.Search.Documents.Indexes.Models.LexicalAnalyzerName analyzerName) { }
+        public AnalyzeTextOptions(string text, Azure.Search.Documents.Indexes.Models.LexicalNormalizerName normalizerName) { }
```

Comment:
I can't use an analyzer and a normalizer together?

Reply:
I tried the other combination and got an error -
{"error":{"code":"InvalidRequestParameter","message":"The request is invalid.","details":[{"code":"InvalidAnalyzeRequest","message":"**The 'normalizer' and the 'tokenizer' properties can't be set at the same time.** Parameters: analyzeRequest"}]}}

Reply:
You should try analyzers too because [the docs](https://docs.microsoft.com/en-us/azure/search/search-normalizers#predefined-and-custom-normalizers) only say:

> (1) Custom normalizers do not specify tokenizers since normalizers always produce a single token.

Reply:
I looked in the service code and all 3 are mutually exclusive. No 2 can be passed together.

Reply:
Here's the [service code](https://msdata.visualstudio.com/Azure%20Search/_git/AzureSearch?path=%2FSource%2FSearch%2FProduct%2FSearchCore%2FModels%2FAnalyzeRequest.cs&version=GBcurrent&line=137&lineEnd=148&lineStartColumn=1&lineEndColumn=14&lineStyle=plain&_a=contents) snippet -

```
            if (!Analyzer.IsNullOrEmpty() && !Tokenizer.IsNullOrEmpty())
            {
                return IncompatibleAnalysisComponentValidationResult(analyzerParam, tokenizerParam);
            }
            if (!Normalizer.IsNullOrEmpty() && !Tokenizer.IsNullOrEmpty())
            {
                return IncompatibleAnalysisComponentValidationResult(normalizerParam, tokenizerParam);
            }
            if (!Analyzer.IsNullOrEmpty() && !Normalizer.IsNullOrEmpty())
            {
                return IncompatibleAnalysisComponentValidationResult(analyzerParam, normalizerParam);
            }
```